src/variables/components/TypeAheadVariableDropdown.test.tsx

```diff
@@ -9,37 +9,43 @@ import TypeAheadVariableDropdown from 'src/variables/components/TypeAheadVariabl
 import {renderWithRedux} from 'src/mockState'
 import {AppState, RemoteDataState} from 'src/types'
 
-const values = {
-  def: 'defbuck',
-  def2: 'defbuck2',
-  foo: 'foobuck',
-  goo: 'goobuck',
-  new: 'newBuck',
-  always: 'always',
-  really: 'reallyYes',
-  REALLy2: 'anotherReally',
+// map variable name
+const variable_name = 'xkcd_meme'
+
+/* Note(sahas): Note that there are only 5 values in the variable for the purposes of this unit test.
```

Comment:
Thanks again - I think this is a useful comment because folks (who might not realize this is using react-window) may otherwise be confused later about why all dropdown items aren't being rendered.